src/Orchard.Web/Modules/Orchard.Layouts/Views/EditorTemplates/LayoutEditor.cshtml

```diff
@@ -26,9 +26,6 @@
     Style.Include("LayoutEditor.css", "LayoutEditor.min.css");
     Script.Include("LayoutDesignerHost.js");
 
-    // The grid system.
-    Style.Include("default-grid.css");
```

Comment:
This removes probably extensibility.

Reply:
This is only in the editor shape for layouts.
I think for front-end you are referring to what's in TheThemeMachine theme? That's a separate css file (with the same name and perhaps same content) that is included in the layout there.

Here in the backoffice editor for the layout, the content of that stylesheet "conflicts" with bootstrap's grid system. I would argue that the stylesheet itself should also be removed from Orchard.Layouts.
In case someone would want to extend styling for this editor, as it is, they would have to go in the sources and change that stylesheet, and at that point one might as well add a `Style.Include("my-new.css");` line in LayoutEditor.cshtml.

Reply:
Should not be removed from Orchard.Layouts, because that's the default to use when it's not themed. If there is a conflict on the admin, then we need to ensure that all existing classes are also in bootstrap. Or change the names of the classes to they don't collide.